dev-util/hip/files/hip-4.2.0-config-cmake-in.patch

```diff
@@ -0,0 +1,25 @@
+diff --color -uprN orig/hip-config.cmake.in HIP-rocm-4.2.0/hip-config.cmake.in
```

Comment:
I know this is adopted from hip-3.5.1-config-cmake-in.patch, but please also add a patch header to explain the logic behind it.